julia/Related/src/Related.jl

```diff
@@ -98,4 +98,6 @@ function main()
     end
 end
 
+main()
```

Comment:
I tried on the VM, this is the only way to precompile without cause performance loss